libs/gi/db/src/Database/DataManagers/TeamDataManager.ts

```diff
@@ -30,22 +31,75 @@ export class TeamDataManager extends DataManager<
       if (key.startsWith('team_') && !this.set(key, {}))
         this.database.storage.remove(key)
   }
+  newName() {
+    const existing = this.values
+    for (let num = existing.length + 1; num < existing.length + 100; num++) {
```

Comment:
This loop makes no sense, I assume `num` is meant to be used in some way?

Reply:
If so, 
```suggestion
    for let num = 1; num <= existing.length; num++) {
      const name = `Team Name ${existing.length + num}`
```
is better